### ISSUE TITLE
Nim constructor not found for class BulletPhysicsDirectSpaceState

### ISSUE THREAD

**Issue by geekrelief** [pragmagic/godot-nim#102]
Using Bullet, on a Spatial object I'm trying to do a ray cast with `PhysicsDirectSpaceState`
```nim
  var spaceState = self.getWorld().directSpaceState
```
The above generates an error from `godotnim.nim`.  `asNimGodotObject` calls `newNimGodotObject` with the name returned from `godotinternal.nim#getClassNameRaw()`.

`BulletPhysicsDirectSpaceState` is not part of the api.json.

This isn't the fault of godot-nim.  The godot engine doesn't export it properly.  The work around is modify `godottypes.nim` with a new type and register the class.
```nim
BulletPhysicsDirectSpaceState* = ref object of PhysicsDirectSpaceState

...

registerClass(BulletPhysicsDirectSpaceState, "BulletPhysicsDirectSpaceState", true)
```

I've bugged this https://github.com/godotengine/godot/issues/48752 and created an engine PR to fix.

**Comment by endragor** [comment]
Yes, this basically duplicates #91. Thanks for submitting the PR to Godot.
